### PR TITLE
[utils] Add error handling in csvfile validation

### LINKED ISSUE #251: [bug] Form error on "Add batch user creation"
**Error:**
```
UnicodeDecodeError at /admin/openwisp_radius/radiusbatch/add/
'utf-8' codec can't decode byte 0xa1 in position 11: invalid start byte
```

**How to reproduce:**
1. Go to `<domain>/admin/openwisp_radius/radiusbatch/add/`
2. Select "Strategy:" as "Import from CSV"
3. Upload a PDF file instead of CSV.

**Expected Behaviour:**
Graceful handling of error; telling user that incorrect format file has been uploaded and asking them to try again.

### PR DESCRIPTION
**Pull request by Aviral14**
Fixes #251

### CODE REVIEW

**Comment by Aviral14** [comment]
@atb00ker All checks have passed now!

File: django_freeradius/tests/base/test_utils.py [hunk 1]
```diff
@@ -7,17 +7,24 @@
...
+        self.assertEqual(find_available_username("rohith", []), "rohith1")
+        User.objects.create(username="rohith1", password="password")
+        self.assertEqual(find_available_username("rohith", []), "rohith2")
 
     def test_validate_csvfile(self):
-        invalid_csv_path = self._get_path('static/test_batch_invalid.csv')
-        improper_csv_path = self._get_path('static/test_batch_improper.csv')
+        invalid_csv_path = self._get_path("static/test_batch_invalid.csv")
+        improper_csv_path = self._get_path("static/test_batch_improper.csv")
+        invalid_format_path = self._get_path("static/test_batch_invalid_format.pdf")
```
Comment (nemesisdesign) [inline django_freeradius/tests/base/test_utils.py]: I think this test method is becoming too big, can you please write a new test method with the assertions you have added?

Another suggestion: please keep the changes to minimum, do not edit lines that are not bugged, right now I see you're changing a lot of single quotes to double quotes, which even if it's good for consistency, adds noise and makes the work of the reviewers harder. Those kind of clean up changes should be done separately if you really want to do it (clean up is welcome in my opinion).

**Comment by nemesisdesign** [comment]
PS: please also rebase your branch on the current master.

File: django_freeradius/tests/base/test_utils.py (hunk 1 above)
Comment (Aviral14) [reply django_freeradius/tests/base/test_utils.py]: @nemesisdesign ok I will try to write a completely new method for testing the validation of csv files.
Actually,my formatter was automatically doing these cleanup changes you mentioned.I will remember not to do any kind of edits in non-buggy lines!

**Comment by nemesisdesign** [comment]
@Aviral14 ping me when ready for review

**Comment by Aviral14** [comment]
@nemesisdesign Please Review!

**Pull request merged**